hw/mcu/nordic/nrf5340/nrf5340.ld

```diff
@@ -152,6 +152,10 @@ SECTIONS
         *(.data*)
 
         *(.jcr)
+
+        __MYNEWT_SENSITIVE_START = .;
+        *(.mynewt_sensitive*)
+        __MYNEWT_SENSITIVE_END = .;
```

Comment:
So, I think these should be consistent with the other variables, so, can you please make it lowercase.

Reply:
changed these to lower case